### DISCUSSION TITLE
jetbrains support

### DISCUSSION THREAD

**Discussion by win4r** [cline/cline#501] [Feature Requests]
https://youtu.be/7Y8Q5IcOey8
jetbrains support

**Comment by Joseph-Cardwell** [comment]
> https://youtu.be/7Y8Q5IcOey8 jetbrains support

this has nothing to do with the topic and has been reported to github and youtube as against the TOS.

**Comment by LousyBook-01** [comment]
yeah, i would like jetbrains support as well, but @Joseph-Cardwell why are you reporting this? is it bcuz of the video?

**Reply by Joseph-Cardwell** [reply]
yes

**Comment by cantalupo555** [comment]
It would be awesome to have Cline in Jetbrains IDE.

**Reply by LousyBook-01** [reply]
> It would be awesome to have Cline in Jetbrains IDE.

That's what i was saying, it would be awesome, i only need it for "Fleet", "PyCharm", and "Rider" tho, it has better support for languages than vscode

**Comment by ahmetegesel** [comment]
only reason I am still not using cline. Please bring jetbrains support, it would be marvellous.

**Comment by d121122gma** [comment]
> https://youtu.be/7Y8Q5IcOey8
> 
> jetbrains support

**Comment by dcbartlett** [comment]
Moving discussion to https://github.com/cline/cline/discussions/581
